openslides_backend/action/actions/motion/json_upload.py

```
@@ -0,0 +1,708 @@
+                None,
+                category_id,
+                other_forbidden_numbers=self._previous_numbers,
+            )
+            if number := value.get("number"):
+                entry["number"] = {"value": number, "info": ImportState.GENERATED}
+                self._previous_numbers.append(number)
+
+        has_submitter_error: bool = False
+        for fieldname in ["submitter", "supporter"]:
+            if users := entry.get(f"{fieldname}s_username"):
+                if not isinstance(users, list):
+                    users = [users]
+                verbose = entry.get(f"{fieldname}s_verbose", [])
+                if not isinstance(verbose, list):
+                    verbose = [verbose]
+                verbose_user_mismatch = len(verbose) > len(users)
+                username_set: Set[str] = set([])
+                entry_list: list[Dict[str, Any]] = []
+                message_set = set()
+                lookup = (
+                    self.submitter_lookup
+                    if fieldname == "submitter"
+                    else self.supporter_lookup
+                )
+                for user in users:
+                    if isinstance(user, str):
+                        if verbose_user_mismatch:
+                            entry_list.append(
+                                {"value": user, "info": ImportState.ERROR}
+                            )
+                        elif user in username_set:
+                            entry_list.append(
+                                {"value": user, "info": ImportState.WARNING}
+                            )
+                            message_set.add(
+                                f"At least one {fieldname} has been referenced multiple times"
+                            )
+                        else:
+                            username_set.add(user)
+                            found_users = lookup.get(user, [])
```

You cannot get more than one user, the username is unique. If you find duplicate users, you should throw an exception and finish the json_upload immediately,

---

Done